### DISCUSSION TITLE
[Feature request] Control speech rate for XTTS V2

### DISCUSSION THREAD

**Discussion by hengjiUSTC** [coqui-ai/TTS#3274] [General Q&A]
Like the functionalities in Coqui studio, how could we control speech rate for XTTS v2 through python api?

**Comment by hengjiUSTC** [comment]
I use the configuration on hugging face https://huggingface.co/coqui/XTTS-v2/blob/main/config.json, but the result quality get locally is not as good as Coqui studio. Is there other configuration we need to use for Chinese language?

**Comment by zyb8543d** [comment]
I also encountered the same problem. The pause was unnatural and the local effect was worse than huggingface demo.

**Comment by erogol** [comment]
Questions go to discussions.

**Comment by polvallverdu** [comment]
Would be great to know! Did any of you find any setting that can change that?

**Comment by jackyin68** [comment]
Maybe some problem related to sample rate

**Reply by zyb8543d** [reply]
Impossible because we used the official example

**Comment by hengjiUSTC** [comment]
find this: https://huggingface.co/coqui/XTTS-v2/discussions/1
seems not supported yet?
